README.md

```diff
@@ -103,33 +127,24 @@ to use these snapshots in your build, you'll need to add the JitPack repository
 
    then
 
-   ```
+   ```SBT
    <dependency>
      <groupId>io.github.spark-redshift-community</groupId>
-     <artifactId>spark-redshift_2.10</artifactId>  <!-- For Scala 2.11, use spark-redshift_2.11 instead -->
-     <version>4.0.0-SNAPSHOT</version>
+     <artifactId>spark-redshift</artifactId>
+     <version>master-SNAPSHOT</version>
    </dependency>
    ```
 
 - **In SBT**:
-   ```
+   ```SBT
    resolvers += "jitpack" at "https://jitpack.io"
    ```
 
    then
 
+   ```SBT
+   libraryDependencies += "io.github.spark-redshift-community" %% "spark-redshift" % "master-SNAPSHOT"
    ```
-   libraryDependencies += "io.github.spark-redshift-community" %% "spark-redshift" % "4.0.0-SNAPSHOT"
-   ```
-
-- In Databricks: use the "Advanced Options" toggle in the "Create Library" screen to specify
-  a custom Maven repository:
-
-  ![](https://cloud.githubusercontent.com/assets/50748/20371277/6c34a8d2-ac18-11e6-879f-d07320d56fa4.png)
-
-  Use `https://jitpack.io` as the repository.
-
-  - For Scala 2.11: use the coordinate `io.github.spark-redshift-community:spark-redshift_2.11:master-SNAPSHOT`
```

Comment:
If I understand correctly, Databricks users have no need for this fork, so there's no reason to keep this in the README.